components/Fields/OpeningHours.test.js

```diff
@@ -1,83 +0,0 @@
-import { createPinia } from 'pinia'
```

Comment:
This file should really be removed ?

Reply:
I think it's not used anymore, we never play this test ... and it triggers lint errors (that we could fix of course).
What do you think ?

Reply:
I think it was never used automatically. But the goal is to have more tests, not less.

Reply:
Alright, I agree. I've fixed the lint errors !